test/test_opsum_to_ttn.jl

```diff
@@ -1,12 +1,23 @@
 using Dictionaries
 using ITensors
+using ITensorGaussianMPS: hopping_hamiltonian
 using ITensorNetworks
 using Random
+using LinearAlgebra: eigvals
 using Test
 
+function to_matrix(t::ITensor)
+  c = combiner(inds(t; plev=0))
+  tc = (t * c) * dag(c')
+  cind = combinedind(c)
+  return matrix(tc, cind', cind)
+end
+
 @testset "OpSum to TTN converter" begin
   @testset "OpSum to TTN" begin
     # small comb tree
+
+    ITensors.disable_auto_fermion()
```

Comment:
I guess this is to circumvent the bug you mentioned? Would be good to leave a comment about that, so we know to remove this line when that bug is fixed.